portfolio/src/main/java/com/google/sps/servlets/ListComment.java

```diff
@@ -51,18 +53,20 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         response.getWriter().println(json);
     }
 
-    @Override
-    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-        String comment = request.getParameter("comment");
-        long timestamp = System.currentTimeMillis();
-
-        Entity commentEntity = new Entity("Comment");
-        commentEntity.setProperty("comment", comment);
-        commentEntity.setProperty("timestamp", timestamp);
+    private int getCommentCount(HttpServletRequest request){
+        String numCommentString = request.getParameter("num");
 
-        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-        datastore.put(commentEntity);
+        int numComments = 10;
+        try{
+            numComments = Integer.parseInt(numCommentString);
+        } catch(NumberFormatException e){
```

Comment:
nit space after `catch` and after closing parenthesis
```suggestion
        } catch (NumberFormatException e) {
```